apps/dashboard/.env

```diff
@@ -34,3 +34,6 @@ DIRECT_DATABASE_CONNECTION_STRING=# enter your direct (unpooled or session mode)
 # Misc, optional
 STACK_ACCESS_TOKEN_EXPIRATION_TIME=# enter the expiration time for the access token here. Optional, don't specify it for default value
 NEXT_PUBLIC_STACK_HEAD_TAGS=[{ "tagName": "script", "attributes": {}, "innerHTML": "// insert head tags here" }]
+
+#frontend url
+DASHBOARD_BASE_URL="https://stack-auth.com/"
```

Comment:
There is already a variable `NEXT_PUBLIC_STACK_URL`

Reply:
i have change env variable